packages/ui-components/src/components/action-button-icon/action-button-icon.tsx

```diff
@@ -1,17 +1,17 @@
 import { Component, Event, EventEmitter, h, Host, Prop } from '@stencil/core';
-import { EActionButtonType, IButton, IButtonEvents } from '../action-button/action-button.types';
-import { EAnchorTarget, EComponentSize, IAnchor } from '../../utils/types';
+import { EActionButtonType } from '../action-button/action-button.types';
+import { EAnchorTarget, EComponentSize } from '../../utils/types';
 import { EIconName, EOtherIconName } from '../icon/icon.types';
-import { IButtonIcon } from './action-button.types';
 import { EBadgeState } from '../badge/badge.types';
 import { isEmpty } from 'lodash-es';
+import { IActionButtonTextConfig } from './action-button-icon-types';
 
 @Component({
 	tag: 'kv-action-button-icon',
 	styleUrl: 'action-button-icon.scss',
 	shadow: true
 })
-export class KvActionButtonIcon implements IButton, IButtonIcon, IButtonEvents, IAnchor {
+export class KvActionButtonIcon implements IActionButtonTextConfig {
 	/** (required) Button's icon symbol name */
```

Comment:
You don't need these doc blocks here because they are inherited from the interface.

Reply:
you might need to use `/** @inheritdoc */`